docker-compose.yml

```diff
@@ -951,6 +952,21 @@ services:
   talk-janus:
     image: ghcr.io/juliushaertl/nextcloud-dev-talk-janus:latest
 
+  talk-recording:
+    image: nextcloud/aio-talk-recording:latest
+    environment:
+      NC_DOMAIN: "nextcloud${DOMAIN_SUFFIX}"
+      ALLOW_ALL: true
```

Comment:
@SystemKeeper for me this causes:

ERROR: The Compose file './docker-compose.yml' is invalid because:
services.talk-recording.environment.ALLOW_ALL contains true, which is an invalid type, it should be a string, number, or a null